generators/search-map.js

```diff
@@ -25,17 +25,19 @@ module.exports = function(docMap, siteConfig) {
 	for (name in docMap) {
 		if (docMap.hasOwnProperty(name)) {
 			var docObj = docMap[name];
-			
-			var description = helpers.stripMarkdown(docObj.description);
-			description = unescapeHTML(description);
+			// If there is no body, it's likely we don't want to index it
+			if(docObj.description || docObj.signatures || docObj.params || docObj.return || docObj.options){
```

Comment:
I believe that's normal for a module's `docObject`. I thought the goal was to just make sure one of those params was defined because any of them could be content.